Libraries/LibWeb/MixedContent/AbstractOperations.cpp

```diff
@@ -22,7 +26,7 @@ void upgrade_a_mixed_content_request_to_a_potentially_trustworthy_url_if_appropr
         // 2. request’s URL’s host is an IP address.
         || (request.url().host().has_value() && (request.url().host()->has<URL::IPv4Address>() || request.url().host()->has<URL::IPv6Address>()))
 
-        // 3. § 4.3 Does settings prohibit mixed security contexts? returns "Does Not Restrict Mixed Security Contents" when applied to request’s client.
+        // 3. § 4.3 Does settings prohibit mixed security contexts? returns "Does Not Restrict Mixed Security Contents" when applied to request’s client.
```

Comment:
Apparently this spec has some non-breaking spaces in it when you copy from it, caught my eye in the IDE.